modules/animation/src/animationcontroller.cpp

```diff
@@ -425,6 +423,6 @@ Seconds AnimationController::deltaTime() const {
 
 Seconds AnimationController::getCurrentTime() const { return currentTime_; }
 
-}  // namespace animation
```

Comment:
Is this a new format setting?
Personally, I think that there should be a space

Reply:
yes, I have added it back. had some local version update of clang-format which did the wrong thing...